forge-game/src/main/java/forge/game/card/CardState.java

```diff
@@ -720,6 +738,7 @@ public final void copyFrom(final CardState source, final boolean lki, final Card
         // Makes a "deeper" copy of a CardState object
         setName(source.getName());
         setType(source.type);
+        this.changedType = source.changedType;
```

Comment:
is this line for LKI?
but it would be wrong for normal copy effects to just keep all changes...

Reply:
@Hanmac
no, you mean `Card.copyFrom`